sql/src/main/java/io/crate/operation/scalar/arithmetic/CeilFunction.java

```diff
@@ -59,7 +59,7 @@ public Symbol normalizeSymbol(Function symbol) {
     static class DoubleCeilFunction extends CeilFunction {
 
         private static final FunctionInfo INFO = new FunctionInfo(
-                new FunctionIdent(NAME, ImmutableList.<DataType>of(DataTypes.DOUBLE)), DataTypes.LONG);
+                new FunctionIdent(NAME, ImmutableList.<DataType>of(DataTypes.DOUBLE)), DataTypes.LONG, FunctionInfo.Type.SCALAR, true, true);
```

Comment:
as you described in the preservesComparisonRelation javadoc, ceil does not preserves comparison